src/01-nuclear-energetics.md

```diff
@@ -0,0 +1,433 @@
+daughter_NZ.head(10)
+```
+
+Some of the rows in the above table don't make sense because they are either both zero or contain negative numbers. The first row that does make sense is row number 9.
+
+Although we can tell by eye that the daughter element corresponding the row 9 with N=1 and Z=0 is the neutron, in general we will need to query the iaea table to find this out. This querying is similar to what we did earlier, the only difference is that below we now use [fstrings](https://realpython.com/python-f-strings/) to demonstrate how we go about removing the hard coded numbers.
+
+```python
+q = f"N=={daughter_NZ.loc[9]['N']} & Z=={daughter_NZ.loc[9]['Z']}"
+q
+```
+
+```python
+iaea.query(q)
+```
+
+We are now ready to automate the process of finding the daughter elements (for those that have one).
+
+```python
+parent_index = []
+daughter_index = []
+for i, row in daughter_NZ.iterrows():
+    try:
+        q = f"N=={row['N']} & Z=={row['Z']}"
+        daughter_index.append(iaea.query(q).index[0])
+        parent_index.append(i)
+    except:
+        continue
+```
+
+```python
+parents_alpha_decay = iaea.loc[parent_index]
+parents_alpha_decay.reset_index(inplace=True, drop=True)
+
+daughters_alpha_decay = iaea.loc[daughter_index]
+daughters_alpha_decay.reset_index(inplace=True, drop=True)
+```
+
+```python
+parents_alpha_decay.head()
+```
+
+```python
+daughters_alpha_decay.head()
+```
+
+The above two tables pair the parents and daughters together. We can now calculate the kinetic energy of the hypothetical decay reactions.
+
+```python
+parents_alpha_decay["E_kin (MeV)"] = (parents_alpha_decay["Mass (u)"] - 
+                      (daughters_alpha_decay["Mass (u)"] + alpha["Mass (u)"].values[0]))*u
+```
+
+```python
+parents_alpha_decay.head(10)
+```
+
+```python
+parents_alpha_decay.plot.scatter(x="A", y="E_kin (MeV)",figsize=(15,8), 
+                                 title="Kinetic energy of alpha decay products     (Fig 3)");
+plt.plot((270, 0), (0, 0), 'r-');
+```
+
+Fig 3 shows us that on the whole (with the exception of He-5, Li-5 and Be-8) spontaneous alpha decay is only energetically possible when the mass number gets higher than about 100. We can see this explicitly by querying the `parents_alpha_decay` table
+
+```python
+parents_alpha_decay.query("`E_kin (MeV)` > 0")
+```
+
+We can also see that if we are able to deposit more energy (in MeV) than
+
+```python
+abs(parents_alpha_decay["E_kin (MeV)"].min())
+```
+
+Then, energetically speaking, alpha decay is possible for all elements.
+
+
+Some elements, require a lot less than 25MeV. For example, we can "chain" together the query function to the plotting function in order to conveniently pick out  [Palladium](https://en.wikipedia.org/wiki/Palladium) and [Silver](https://en.wikipedia.org/wiki/Silver)
+
+```python
+fig, axes = plt.subplots(nrows=1, ncols=2, figsize=(15,8), sharey=True)
+
+parents_alpha_decay.query("EL=='Pd'").plot.scatter(x="A", y="E_kin (MeV)", ax=axes[0],
```

Comment:
Maybe call the x-axis Pd-A? It took me a moment to understand that this graph is all isotopes from one element. (it's obvious once realized)

Reply:
I can't change the x-axis label easily because `plot` pulls the label from the data frame. I have instead added a legend which I hope will do the trick.